### ISSUE TITLE
WebSocket opening handshake timed out

### ISSUE THREAD

**Issue by bORm** [balderdashy/sails#2682]
sometimes pops in console
WebSocket connection to 'ws://MYSITE.COM:1337/socket.io/?__sails_io_sdk_version=0.11.0&__sails_io_sdk_platform=browser&__sails_io_sdk_language=javascript&EIO=3&transport=websocket&sid=GhYMY2AldnLIRfuQAAAR' failed:

**Comment by sgress454** [comment]
@bORm without more information it's kinda hard to tell whether this is a Sails issue or not!

**Comment by bORm** [comment]
That happen when i change tab in browser when opened sails application to another and after ( any time ) change to sails tab again
browser Google Chome

**Comment by sgress454** [comment]
Yeah, that sounds like a browser issue; changing tabs back and forth shouldn't cause sockets to attempt to reconnect.  Do you have any browser extensions on?

**Comment by bORm** [comment]
Adblock Plus 1.8.11
Google Mail Checker 4.4.0
JSONView 0.0.32.2
Window Resizer 1.9.1.0

**Comment by sailsbot** [comment]
Thanks for posting, @bORm.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by bORm** [comment]
Problem resolved?
